source/geod24/LocalRest.d

```diff
@@ -390,131 +655,175 @@ public final class RemoteAPI (API) : API
         }
     }
 
+
     /***************************************************************************
 
         Main dispatch function
 
-       This function receive string-serialized messages from the calling thread,
-       which is a struct with the sender's Tid, the method's mangleof,
-       and the method's arguments as a tuple, serialized to a JSON string.
-
-       `geod24.concurrency.receive` is not `@safe`, so neither is this.
+        This function receive string-serialized messages from the calling thread,
+        which is a struct with the sender's LocalTransceiver, the method's mangleof,
+        and the method's arguments as a tuple, serialized to a JSON string.
 
-       Params:
-           Implementation = Type of the implementation to instantiate
-           args = Arguments to `Implementation`'s constructor
+        Params:
+            Implementation = Type of the implementation to instantiate
+            args = Arguments to `Implementation`'s constructor
 
     ***************************************************************************/
 
-    private static void spawned (Implementation) (C.Tid self, CtorParams!Implementation cargs)
+    private static LocalTransceiver spawned (Implementation) (CtorParams!Implementation cargs)
```

Comment:
You changed this function to spawn the thread, instead of being the entry point of the Thread.
While that can be fine, there's a big issue with the `CtorParams`: They cannot be passed like this if they are not `shared`.
The original call used `spawn` which did the check. However passing a delegate bypasses it.

Reply:
I'll use `isSpawnable` to handle it.
That'll do, right?

Reply:
I modified it using `hasLocalAliasing`. Please make sure you want this.